5.0.20 not installable

- [x] I've asked for help in the [Truffle Gitter](http://gitter.im/Consensys/truffle) before filing this issue.

---------------------------

## Issue

Latest 5.0.20 is not installable. Tested on mac and linux. Error is similar:
```
npm install -g truffle
/usr/local/bin/truffle -> /usr/local/lib/node_modules/truffle/build/cli.bundled.js

truffle@5.0.20 postinstall /usr/local/lib/node_modules/truffle
node ./scripts/postinstall.js

/usr/local/lib/node_modules/truffle/scripts/postinstall.js:22
throw error;
^

Error: Error while attempting to download and cache solc 0.5.8: Command failed: node ./build/cli.bundled.js obtain --solc=0.5.8
at postinstallObtain (/usr/local/lib/node_modules/truffle/scripts/postinstall.js:13:11)
at Object.<anonymous> (/usr/local/lib/node_modules/truffle/scripts/postinstall.js:20:3)
at Module._compile (module.js:624:30)
at Object.Module._extensions..js (module.js:635:10)
at Module.load (module.js:545:32)
at tryModuleLoad (module.js:508:12)
at Function.Module._load (module.js:500:3)
at Function.Module.runMain (module.js:665:10)
at startup (bootstrap_node.js:187:16)
at bootstrap_node.js:608:3
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! truffle@5.0.20 postinstall: node ./scripts/postinstall.js
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the truffle@5.0.20 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR! /Users/info/.npm/_logs/2019-06-01T11_57_43_867Z-debug.log
```
5.0.19 works fine on both systems. 5.0.20 fails on the same 2 systems.

Mac: 10.13.6
npm --version
6.4.1
node --version
v8.7.0

--- a/packages/truffle/scripts/postinstall.js
+++ b/packages/truffle/scripts/postinstall.js
@@ -19,5 +19,5 @@ const postinstallObtain = () => {
 try {
   postinstallObtain();
 } catch (error) {
-  throw error;
+  console.error(error);
 }

